### PR TITLE
adds integration tests

### PR DESCRIPTION
**Pull request by minaorangina**
Tests the publish api

### CODE REVIEW

**Comment by minaorangina** [comment]
screen shot 2019-02-12 at 16 00 02

File: __tests__/integration/publish.test.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    const instanceId = '25c7b7c2-cabc-4e6b-b7d0-c9ad5c787e50';
+    const secretKey =
+        'A2DB402A49D7F7F45044D2049F5B2CDEB793564296B396A9B15A727ABE93EB50';
+    let baseUrl = `https://${instanceId}.pushnotifications.pusher.com/publish_api/v1/instances/${instanceId}/publishes`;
+
+    describe('Interests', () => {
+        let baseRequest;
+
+        beforeEach(() => {
+            baseRequest = request.defaults({
```
Comment (luismfonseca) [inline __tests__/integration/publish.test.js]: The idea is not to create a raw request here, but rather to use the library as a customer would

File: __tests__/integration/publish.test.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            return baseRequest(options).then(res => {
+                expect(res.body).toHaveProperty(
+                    'publishId',
+                    expect.any(String)
+                );
+            });
+        });
+    });
+
+    describe('Users', () => {
```
Comment (luismfonseca) [inline __tests__/integration/publish.test.js]: Missing the `deleteUser`

File: __tests__/integration/publish.test.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            }
+        };
+
+        return expect(
+            beamsInstance.publishToUsers(['user0001'], body)
+        ).resolves.toHaveProperty('publishId', expect.any(String));
+    });
+});
+
+describe('deleteUsers', () => {
```
Comment (luismfonseca) [inline __tests__/integration/publish.test.js:47]:
```suggestion
describe('deleteUser', () => {
```

**Review by luismfonseca — APPROVED** [review]
Great! 👍

**Pull request merged**